Bash_scripts/HW21/in_hw20/group_vars/ubuntu/ubuntu_var.yml

```diff
@@ -0,0 +1,4 @@
+#Ubuntu vars
+ansible_user: vagrant
+ansible_ssh_private_key_file: /home/engelko/.ssh/id_rsa
```

Comment:
Do you need this var here if you already have it under linux group?

Reply:
Sorry, this is not the job, this is hw21, everything is fixed, the directory is deleted